docs/website/docs/dlt-ecosystem/verified-sources/filesystem/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Filesystem & Buckets
+description: dlt verified source for Filesystem & Buckets
+keywords: [readers source and filesystem, files, filesystem, readers source, cloud storage]
+---
+
+The Filesystem source is a generic source that allows seamless loading files from the following locations:
```

Comment:
btw i'm also not a huge fan of the phrase "generic source" - what's generic about it ?